Solutions/NozomiNetworks/Data Connectors/Connector_CEF_NozomiNetworks.json

```diff
@@ -0,0 +1,121 @@
+{
+    "id": "NozomiNetworks",
+    "title": "Nozomi Networks",
+    "publisher": "Nozomi Networks",
+    "descriptionMarkdown": "The [Nozomi Networks](https://www.nozominetworks.com/) data connector provides the capability to ingest Nozomi Networks Events into Azure Sentinel. Refer to the Nozomi Networks PDF documentation for more information.",
```

Comment:
Include the link to Nozomi PDF Documentation here.

Reply:
No direct link to the pdf doc. User can find them after login to the account